### ISSUE TITLE
First user interaction on multiplayground on iOS triggers angry clacking sounds

### ISSUE THREAD

**Issue by gonfunko** [RaspberryPiFoundation/blockly#5798]
**Describe the bug**
The first user interaction (tap) on the multiplayground (https://google.github.io/blockly/tests/multi_playground.html) triggers a bunch of angry clacking sounds.

**To Reproduce**
1. Visit https://google.github.io/blockly/tests/multi_playground.html on an iOS device
2. Tap on a flyout category
3. Listen to the angry clacking

**Expected behavior**
The first user interaction is silent.

**Device**
 - Device: 11" iPad Pro
 - OS: iOS 15.1
 - Browser: Safari

**Comment by truongductri01** [comment]
@gonfunko is this still an issue to be fixed. I am not hearing any sound from my iPad although running IOS 13. It is gen 7th

**Comment by gonfunko** [comment]
Yes, I can still repro this on an iPhone running iOS 17.1.2. It's possible this is due to an iOS change introduced in iOS 14 or 15, but it would be nice to fix regardless.
